tests/unit/lms/tasks/email_digests_test.py

```diff
@@ -28,15 +31,19 @@ def test_it_sends_digests_for_instructors(
 
         assert send_instructor_email_digests.apply_async.call_args_list == [
             call(
-                [[user.h_userid for user in participating_instructors[:3]]],
+                (),
                 {
+                    "h_userids": [
+                        user.h_userid for user in participating_instructors[:3]
+                    ],
```

Comment:
Lots of assertions had to be updated in the tests because the calls now use kwargs